NEWS

```diff
@@ -1,6 +1,14 @@
 Unreleased
 ==========
 
+0.40.0
+======
+
+* [fix] Fix ssl test when linking against openssl 3.5 (#1034)
+* Drop support Python 3.8 (#1021)
+* [doc] Various doc updates (#981, #1033)
+* [env] Make PyPy job optional (#1035)
```

Comment:
Should probably be "dropped PyPy support"?

Reply:
Indeed, I can rebase over it now that the patch is merged. Let's do that

Reply:
done